src/main/java/fi/okm/jod/yksilo/controller/ehdotus/MahdollisuudetController.java

```diff
@@ -89,7 +90,7 @@ public List<EhdotusDto> createEhdotus(@RequestBody @Valid LuoEhdotusDto ehdotus)
     ids.addAll(tyomahdollisuusIds);
     ids.addAll(koulutusmahdollisuusService.fetchAllIds());
 
-    log.info("Creating a suggestion for tyomahdollisuudet");
+    log.info("Creating a suggestions");
```

Comment:
A minor typo, should probably be "Creating suggestions" 

( I wouldn't actually mind the extra expressiveness of "what suggestions" as in perhaps "Creating a suggestions for tyo/koulutusmahdollisuudet")

Reply:
Changed to "Creating the suggestions".